src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 
 ## Upcoming Release
+* Corrected the URLs to Azure Portal in the results of `Get-AzEnvironment` and `Get-AzContext`. [#15429]
```

Comment:
The first line is not in the scope of this PR. It's carried over by the updated common libarary.